cache.js

```diff
@@ -0,0 +1,66 @@
+const MAX_CACHE_SIZE = 100
+
+const generateDefKey = (data, to) => {
+  if (
+    !data ||
+    data.length === 0 ||
+    (!isArrayBuffer(data) && typeof data !== "string")
+  ) {
+    return;
+  }
+
+  const toStr = !!to ? Buffer.from(to).toString("hex") : "";
```

Comment:
Oh no you should just `return` if `!to`. Deploying a contract is not something we can ABI decode

Reply:
[e07a48f ](https://github.com/GridPlus/eth-lattice-keyring/commit/e07a48f392203cb888338402fe1340530cb0522b)